### PR TITLE
License updates

### LINKED ISSUE #2729: Updating the license attributions
## Description of bug
We have a licenses folder that contains a Licenses.txt file with a list of some licenses (not sorted alphabetically) and links as well as individual files for license attributions. This is inconsistent. 

We have no clear indication of where we use the project we are attributing the license for which makes it hard to maintain we is still used or not.

Also linking to the license is not enough, we need the license text in the project with some licenses.

We had some license attributions for stuff we no longer use (that was fixed in last PR)

I am submitting a PR to fix that right now.

### PR DESCRIPTION
**Pull request by valadas**
closes #2729 

## Summary
See issue for more details. This PR removes the licenses.txt file in favor of one file per license attribution, updates the format with a header that allows us to know where we use it and makes sure we have the whole license text for each attribution.

### CODE REVIEW

File: Website/Licenses/Cake (MIT).txt.resources [hunk 1]
```diff
@@ -0,0 +1,30 @@
+﻿PROJECT: Cake
+
+WHERE WE USE IT: Used at multiple places in our build process, for instance at https://github.com/dnnsoftware/Dnn.Platform/blob/development/build.cake
+
+LICENTE TYPE: MIT
```
Comment (bdukes) [inline Website/Licenses/Cake (MIT).txt.resources]: typo (looks like it's present in all files): LICENTE/LICENSE

File: Website/Licenses/CountryListBox (Public Domain).txt.resources [hunk 2]
```diff
@@ -0,0 +1,9 @@
+PROJECT: CountryListBox
+
+WHERE WE USE IT:
+As part of a reusable control at https://github.com/dnnsoftware/Dnn.Platform/tree/development/DNN%20Platform/Controls/CountryListBox
+As par of SolPart Menu at https://github.com/dnnsoftware/Dnn.Platform/tree/development/DNN%20Platform/Controls/SolpartMenu/CountryListBox
```
Comment (bdukes) [inline Website/Licenses/CountryListBox (Public Domain).txt.resources]:
```suggestion
As part of SolPart Menu at https://github.com/dnnsoftware/Dnn.Platform/tree/development/DNN%20Platform/Controls/SolpartMenu/CountryListBox
```

File: Website/Licenses/Cake (MIT).txt.resources (hunk 1 above)
Comment (valadas) [reply Website/Licenses/Cake (MIT).txt.resources]: Correct, it was copy pasted from a template, I will go correct that

Comment (valadas) [reply Website/Licenses/Cake (MIT).txt.resources]: Thanks for spotting that, it is now corrected.

**Pull request merged**